commit origin add-friend-likes

itWasMistake opened the pull request:
Date: Wed Oct 12 23:11
Теперь пользователи могут добавлять друг друга в друзья! Фильмам можно ставить лайки....

itWasMistake committed:
commit origin add-friend-likes
Date: Wed Oct 12 23:11
Теперь пользователи могут добавлять друг друга в друзья!
Фильмам можно ставить лайки....

avfyodorov reviewed (commented):
Добрый день!
На данный момент есть проблема: Красная сборка тестов. Обязательно нужно добиться, чтобы тесты проходили. В случае затруднений с зелёной сборкой, обращайтесь пожалуйста к наставнику.

itWasMistake committed:
commit origin add-friend-likes
Date: Wed Oct 12 23:11
Теперь пользователи могут добавлять друг друга в друзья!
Фильмам можно ставить лайки....

itWasMistake committed:
wed 19 oct 2022
Теперь код статуса и ошибки возвращаются верно

itWasMistake committed:
wed 19 oct 2022
исправлена логика метода getCommonFriends, тепеь можно найти
общих друзей!
